lib/routes/cockroachlabs/blog.ts

```diff
@@ -0,0 +1,88 @@
+import { Route } from '@/types';
+import ofetch from '@/utils/ofetch';
+import { load } from 'cheerio';
+import cache from '@/utils/cache';
+
+export const route: Route = {
+    name: 'Blogs',
+    maintainers: ['CookiePieWw'],
+    categories: ['programming'],
+    path: '/blog/:category',
+    example: '/cockroachlabs/blog/engineering',
+    parameters: { category: 'Blog category, e.g., engineering' },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['cockroachlabs.com/blog/:category', 'cockroachlabs.com/blog'],
+            target: '/cockroachlabs/blog/:category',
+        },
+    ],
+    handler,
+};
+
+async function handler(ctx) {
+    const category = ctx.req.param('category');
+    const rootUrl = 'https://www.cockroachlabs.com';
+    const currentUrl = `${rootUrl}/blog/${category}`;
```

Comment:
Why enforce `category` when it prevents users from subscribing to recent articles, regardless of category?